cylc/flow/terminal.py

```diff
@@ -285,3 +288,43 @@ def wrapper(*api_args: str) -> None:
                 raise exc from None
         return wrapper
     return inner
+
+
+def prompt(message, options, default=None, process=None):
+    """Dead simple CLI textural prompting.
```

Comment:
```suggestion
    """Dead simple CLI textual prompting.
```

https://www.dictionary.com/browse/textural